pallets/loans/src/lib.rs

```diff
@@ -779,6 +780,29 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Apply a proposed change identified by a change id.
+		/// It will only perform the change if the requirements for it
+		/// are fulfilled.
+		#[pallet::weight(100_000_000)]
+		#[pallet::call_index(10)]
+		pub fn apply_write_off_policy(
```

Comment:
I've added this as a new extrinsic. The signature is the same for both `apply_loan_mutation()` and `apply_write_off_policy()` but having them in separated extrinsic allow to calculate correctly the weights for each case. Otherwise we would need to run the extrinsics and fix the weight which is not desirable for a user to know how many fees will need.

Reply:
Makes sense!